site/profile/manifests/core/x2go_agent.pp

```diff
@@ -0,0 +1,20 @@
+# @summary
+#   Adds additional packages for x2go agent
```

Comment:
This is a trivial class, and we other examples like this in the tree, but FYI, I have been trying to slowly remove them.  In general, application specific class that aren't "composing" should be an external module in the `Puppetfile`.

Reply:
Noted.